src/packages/documents/documents/components/input-document/input-document.element.ts

```diff
@@ -189,7 +193,7 @@ export class UmbInputDocumentElement extends UUIFormControlMixin(UmbLitElement,
 	#renderItem(item: UmbDocumentItemModel) {
 		if (!item.unique) return;
 		return html`
-			<uui-ref-node name=${item.name} id=${item.unique}>
+			<uui-ref-node name=${item.name} id=${item.unique} ?disabled=${this.#isDraft(item)}>
```

Comment:
We should go with a class here, the item is not disabled. If you set the opacity to 0.6 then it's the same as the tree item.